projects/packages/my-jetpack/_inc/components/product-cards-section/protect-card/logins-blocked-status.tsx

```diff
@@ -0,0 +1,130 @@
+import { __ } from '@wordpress/i18n';
+import { useMemo } from 'react';
+import useProduct from '../../../data/products/use-product';
+import { getMyJetpackWindowInitialState } from '../../../data/utils/get-my-jetpack-window-state';
+import useMyJetpackConnection from '../../../hooks/use-my-jetpack-connection';
+import ShieldOff from './assets/shield-off.svg';
+import { InfoTooltip } from './info-tooltip';
+import { useProtectTooltipCopy } from './use-protect-tooltip-copy';
+import type { ReactElement, PropsWithChildren } from 'react';
+
+export const LoginsBlockedStatus = () => {
+	const slug = 'protect';
+	const { detail } = useProduct( slug );
+	const { isPluginActive: isProtectPluginActive = false } = detail || {};
+	const { isSiteConnected } = useMyJetpackConnection();
+	const {
+		protect: { wafConfig: wafData },
+		plugins,
+	} = getMyJetpackWindowInitialState();
+	const { blocked_logins: blockedLoginsCount, brute_force_protection: hasBruteForceProtection } =
+		wafData;
+
+	const isJetpackPluginActive = useMemo( () => {
+		const jetpackPlugin = Object.values( plugins ).find( plugin => plugin?.Name === 'Jetpack' );
+		return jetpackPlugin && jetpackPlugin.active;
+	}, [ plugins ] );
```

Comment:
I wonder if this could be good to move to the utils folder as a reusable function 🤔

Reply:
Yeah, good idea.